provisioning/kubernetes/build.sh

```diff
@@ -1,6 +1,21 @@
 #!/usr/bin/env bash
 set -Eeuo pipefail
 
+# Etcd cluster for Templates API
+: "${ETCD_REPLICA_COUNT:=3}"
+ETCD_ROOT_PASSWORD_BASE64=$(kubectl get secret --namespace default templates-api-etcd -o jsonpath="{.data.etcd-root-password}" 2>/dev/null || echo -e '')
+if [[ "$ETCD_ROOT_PASSWORD_BASE64" == "" ]]; then
+  # Generate random root password if it not set
+  ETCD_ROOT_PASSWORD_BASE64=$(tr -dc A-Za-z0-9 </dev/urandom | head -c 17 | base64 || echo -e '')
+fi
+# Generate list of cluster endpoints
+ETCD_INITIAL_CLUSTER=$(seq 0 $(($ETCD_REPLICA_COUNT-1)) | awk '{ print "templates-api-etcd-"$0"=http://templates-api-etcd-"$0".templates-api-etcd-headless.default.svc.cluster.local:2380"}' | paste -d',' -s)
```

Comment:
Ještě se mu nelíbí ten `paste` 🙂
![CleanShot 2022-08-23 at 10 28 29](https://user-images.githubusercontent.com/215660/186110791-3685c563-da75-4730-85c2-6529c72959b4.jpg)

Reply:
Skus prosim teraz, skusil som tam doplnit medzeru: https://github.com/keboola/keboola-as-code/pull/770/commits/4bcb615cd80fcaec215d951b11aacfd81fb16f21

Reply:
To už jsem zkoušel 😕
![image](https://user-images.githubusercontent.com/215660/186114386-01626812-f813-4f5f-ba30-8634f8771dbb.png)
Koukal jsem do manuálu, ale moc nevím teda. 😕 
![CleanShot 2022-08-23 at 10 45 42](https://user-images.githubusercontent.com/215660/186114607-fb4aa65e-3916-49a2-aa8e-b3c890a4fccf.jpg)

Reply:
asi to na MacOs nepodporuje zretazenie cez `pipe`, nahradim to inym prikazom

Reply:
Mozno to pojde na MacOs takto: https://github.com/keboola/keboola-as-code/pull/770/commits/48475a902c99bf1b46b58217168144fe18743b5f

Reply:
Ajaj, tak ne.
![CleanShot 2022-08-23 at 11 29 36](https://user-images.githubusercontent.com/215660/186123974-22606581-8df6-4b46-97d0-8dd5160befd4.jpg)

Jinak já bych na tom Macu ani tak netrval, ale nejede mi to právě ani v tom Dockeru. 😕
![CleanShot 2022-08-23 at 11 32 18](https://user-images.githubusercontent.com/215660/186124495-653634f0-b409-4bad-8a24-27755a91f905.jpg)

Reply:
zaujimave, lebo mne to prejde :/ 
![image](https://user-images.githubusercontent.com/19371734/186124829-87f9fecc-5197-4bf1-85f8-b4855748f539.png)

Reply:
Mozno mas v tom docker container rozbite opravnenia, ak si to spustil ako root a neskor ako uzivatel:
` docker-compose run --rm -u "$UID:$GID" --service-ports dev bash`

Reply:
Upravil som to - sed bez `-z`: https://github.com/keboola/keboola-as-code/pull/770/commits/f8536bd69e80b850232704520825455d4fd96f4b

Reply:
Funguje 👍